### PR TITLE
chore: pre-release cleanups

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: tools/proto-compiler/src/constants.rs [hunk 1]
```diff
@@ -19,7 +19,8 @@ pub const TENDERMINT_VERSIONS: &[TendermintVersion] = &[
     TendermintVersion {
         repo: "https://github.com/celestiaorg/celestia-core",
         ident: "v0_34",
-        commitish: "v0.34.x-celestia",
+        // TODO: update
+        commitish: "a1b07a1e6c77595466da9c61b37c83b4769b47e7", // "v0.34.x-celestia",
```
Comment (fl0rek) [inline tools/proto-compiler/src/constants.rs:23]: For my curiosity, why was [this particular commit](https://github.com/celestiaorg/celestia-core/commit/a1b07a1e6c77595466da9c61b37c83b4769b47e7) chosen?

**Review by fl0rek — APPROVED** [review]
nice 🎸

File: tools/proto-compiler/src/constants.rs (hunk 1 above)
Comment (zvolin) [reply tools/proto-compiler/src/constants.rs:23]: haha

Comment (zvolin) [reply tools/proto-compiler/src/constants.rs:23]: image

Comment (zvolin) [reply tools/proto-compiler/src/constants.rs:23]: so @oblique entered a branch name there and this is the commit that happen to be there were the current definitions got generated

Comment (zvolin) [reply tools/proto-compiler/src/constants.rs:23]: #1

**Pull request merged**